internal/inventory/tagprocessor_test.go

```diff
@@ -22,6 +22,7 @@ func getTestingLogger() logger.LoggingClient {
 	logLevel := "WARN"
 	if testing.Verbose() {
 		logLevel = "DEBUG"
+		return logger.NewClientStdOut("test", false, logLevel)
 	}
 
 	return logger.NewClientStdOut("test", false, logLevel)
```

Comment:
This line should be using the NewMockClient() so that normally there are no log messages